app/src/main/java/org/simple/clinic/teleconsultlog/shareprescription/TeleconsultSharePrescriptionEffectHandler.kt

```diff
@@ -41,6 +48,15 @@ class TeleconsultSharePrescriptionEffectHandler @AssistedInject constructor(
     }
   }
 
+  private fun loadPatientDetails(): ObservableTransformer<LoadPatientDetails, TeleconsultSharePrescriptionEvent> {
+    return ObservableTransformer { effects ->
+      effects
+          .observeOn(schedulersProvider.io())
+          .map { patientRepository.patientImmediate(it.patientUuid) }
+          .map(::PatientDetailsLoaded)
+    }
+  }
+
```

Comment:
I wanted to re-order it based on what functions comes next in the effectHandler. Should I re-order it for better clarity?

Reply:
Ah, I see now. Not necessary, just that it might sense in the future to split reordering into a different commit so that it's clear what has happened.